index.html

```diff
@@ -182,7 +182,7 @@ <h1>Welcome to the Enhanced Blackbox Explorer!</h1>
                             Export Workspaces...</a>
                         <span class="btn btn-primary btn-file" data-toggle="tooltip" title="Open another log file, video file, exported workspace file or configuration dump file">
                             Open log file/video
-                            <input type="file" class="file-open" multiple>
+                            <input type="file" class="file-open" accept=".bbl,.txt,.cfl,.bfl,.log,.avi,.mov,.mp4,.mpeg,.json" multiple>
```

Comment:
I saw you closed the other input tag `multiple />` but this one not. I don't know if is better to close it or not, both forms are valid (I prefer to close them), but both must be the same. Can you close this tag too?